lib/rules/require-dollar-before-jquery-assignment.js

```diff
@@ -76,9 +76,15 @@ module.exports.prototype = {
             var right;
 
             if (type === 'VariableDeclarator') {
-                left = token.id;
-                varName = left.name;
-                right = token.init;
+                if (token.id.type === 'Identifier') {
+                    left = token.id;
+                    varName = left.name;
+                    right = token.init;
+                } else { // token.id.type === 'ObjectPattern'
+                    // What do I check here?
+                    // token.id.properties is an array both times during the test.
+                    // How to I check first on, then the other?
```

Comment:
I could return maybe, you don't use destructuring for jQuery assignments, do you?